dist/selleckt-legacy.js

```diff
@@ -861,21 +861,38 @@ _.extend(SingleSelleckt.prototype, {
         matchTerm = term.toLowerCase();
 
         return _.reduce(items, function(memo, item){
-            var matchIndex;
+            var matchIndex, matchStart, matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = item.aliases.reduce(function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
```

Comment:
@spmason Yes it will, but do you think it's a problem in this case?

My reasoning was this: pushing anything except `String` or `Number` into aliases leads to "undefined behaviour" anyway. Casting everything to `String` simply avoids throwing an exception. Existing code does not perform any type checks on the `.label` field either.